src/MapsApp.Xamarin.Shared/Views/StartPage.xaml

```diff
@@ -309,24 +292,24 @@
         </Image>
 
         <!-- Activity indicator for long running operations -->
-        <StackLayout Opacity="0.5" Grid.Row="0" Grid.Column="0" Grid.ColumnSpan="2" Grid.RowSpan="4" IsVisible="{Binding IsBusy, Source={StaticResource RouteViewModel}}" BackgroundColor="Gray">
-            <StackLayout Orientation="Vertical" VerticalOptions="CenterAndExpand" HorizontalOptions="CenterAndExpand" BackgroundColor="Gray" >
-                <Label Text="Requesting Route" VerticalOptions="Center" HorizontalOptions="Center" FontSize="Medium" TextColor="White"/>
-                <ActivityIndicator x:Name="AppActivityIndicator" IsRunning="True" Color="White"/>
+        <StackLayout Opacity="0.5" Grid.Row="0" Grid.Column="0" Grid.ColumnSpan="2" Grid.RowSpan="4" IsVisible="{Binding IsBusy, Source={StaticResource RouteViewModel}}" BackgroundColor="{DynamicResource PlaceholderColor}">
+            <StackLayout Orientation="Vertical" VerticalOptions="CenterAndExpand" HorizontalOptions="CenterAndExpand">
+                <Label Text="Requesting Route" VerticalOptions="Center" HorizontalOptions="Center" FontSize="Medium" TextColor="{DynamicResource AccentColor}"/>
+                <ActivityIndicator x:Name="AppActivityIndicator" IsRunning="True" Color="{DynamicResource AccentColor}"/>
             </StackLayout>
         </StackLayout>
 
         <Grid x:Name="NavigationContainerGrid" 
               Grid.Row="0" Grid.RowSpan="4" 
               Grid.Column="0" Grid.ColumnSpan="2"
-              RowSpacing="0" BackgroundColor="White"
+              RowSpacing="0"
               IsVisible="False">
             <Grid.RowDefinitions>
                 <RowDefinition Height="Auto" />
                 <RowDefinition Height="*" />
             </Grid.RowDefinitions>
             <Button Text="Go back" Grid.Row="0" 
-                    BackgroundColor="White" TextColor="Accent" 
+                    TextColor="{DynamicResource AccentColor}" 
                     Clicked="NavigatedView_CloseClicked" />
             <!-- Loading indicator to show when other lists aren't ready yet -->
             <ActivityIndicator Grid.Row="1" IsRunning="True" />
```

Comment:
This indicator displays as too large on Android, filling the whole screen. Setting a width/height for android might fix this.